### ISSUE TITLE
Can not find a piece of example/test/documentation how to call this function. 

### ISSUE THREAD

**Issue by gymad** [dydxprotocol/v3-client#164]
https://github.com/dydxprotocol/v3-client/blob/0f478ad171bee4395785f107df7b0f0f23210e5b/src/modules/private.ts#L440

I get an "Order is not signed and client was not initialized with starkPrivateKey" error message whatever I try to do.

Is there anyone who has a working code for this and is able to share it? so that we can use it as a working example.
Pleaseeee

**Comment by snowkidind** [comment]
Note to authors: there should be a documentation thread that allows people to comment on issues with the documentation

GET v3/orders endpoint is confusing because it says at the top:

"Description: Get active (not filled or canceled) orders for a user by specified parameters"

But later in the doc it provides order statuses for closed orders, making it seem like you can pull for closed orders from this endpoint.

In other words, How does one poll for previously completed orders?

Note: I see that it is possible to poll for fills and retrieve the orders based on fill.orderId using getOrderById

**Comment by snowkidind** [comment]
here's a gist I am working on that will cover what you are asking for in the next cpl days

https://gist.github.com/snowkidind/9545ca43962dc1251a226031ea797b06

**Comment by snowkidind** [comment]
update: gist above is a working example now
